Dockerfile.production

```diff
@@ -0,0 +1,31 @@
+FROM ruby:2.3.3
+MAINTAINER BioSistemika <info@biosistemika.com>
+
+# additional dependecies
+RUN apt-get update -qq && \
+  apt-get install -y \
+  nodejs \
+  postgresql-client \
+  default-jre-headless \
+  sudo graphviz --no-install-recommends \
+  sudo libfile-mimeinfo-perl && \
+  rm -rf /var/lib/apt/lists/*
+
+# install gems
+COPY Gemfile* /tmp/
+WORKDIR /tmp
+RUN bundle install
+
+# create app directory
+ENV APP_HOME /usr/src/app
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+COPY . .
+RUN rm -f $APP_HOME/config/application.yml
+RUN rm -f $APP_HOME/production.env
+
+ENV RAILS_ENV production
+ENV DATABASE_URL=postgresql://postgres@db/scinote_production
+RUN bash -c "PAPERCLIP_HASH_SECRET=$(openssl rand -base64 128 | tr -d '\n') SECRET_KEY_BASE=$(openssl rand -hex 64) rake assets:precompile"
```

Comment:
If I understand this correctly, if we modify this Dockerfile before this line (e.g. add a `apt-get` dependency), that will require the users to re-run `make docker-production`. However, in that case, the 2 variables will be regenerated right? And user will lose his previous hash. I think that's unacceptable, or should at least be mentioned somewhere.

Or am I getting at this all wrong?

In any case, we could maybe add a `make` command to generate those 2 variables? Should be handier than copying the stuff written here/in Wiki.